include/lo2s/perf/event_reader.hpp

```diff
@@ -243,6 +251,15 @@ class EventReader
                     Log::info() << "Lost " << lost->lost << " samples during this chunk.";
                     break;
                 }
+#ifdef HAVE_PERF_RECORD_LOST_SAMPLES
+                case PERF_RECORD_LOST_SAMPLES:
+                {
+                    auto lost = (const RecordLostSamplesType*)event_header_p;
+                    lost_samples += lost->lost;
+                    Log::info() << "Lost " << lost->lost << " samples during this chunk.";
```

Comment:
Since the default logging level is already WARN, I think, this should be at least a warning.